linfa-svm/src/regression.rs

```diff
@@ -43,19 +44,14 @@ pub fn fit_epsilon<F: Float>(
         vec![F::zero(); 2 * target.len()],
         linear_term,
         targets.to_vec(),
+        dataset,
         kernel,
         vec![c; 2 * target.len()],
         params,
         false,
     );
 
-    let mut res = solver.solve();
-
-    for i in 0..target.len() {
-        let tmp = res.alpha[i + target.len()];
-        res.alpha[i] -= tmp;
-    }
-    res.alpha.truncate(target.len());
```

Comment:
This had to be moved inside the solver because the precomputed vector for the linear kernel case had the wrong weights in it. I've now added a regression test to check the correctness of predictions with the precomputed vector too.

Reply:
but how does moving this into the solver changes the solution? I missed a test here :/